src/smdba/postgresqlgate.py

```diff
@@ -137,6 +137,8 @@ def estimate(self):
         """
         Estimate the data.
         """
+
+        PG_VERSION = [int(v_el) for v_el in os.popen("psql --version | sed -e 's/.*\s//g'").read().split('.')]
```

Comment:
So this so called "programming language" does not only command the intendation but also how variable names are written!? And there really are people who like that?